### PR TITLE
Upgraded NetApp SDK to latest version 1.13.0

### PR DESCRIPTION
**Pull request by AdelGhabboun**
1- Upgraded SDK to latest version 1.13.0
2- Activated the cleanup code with a flag
3- Added the waiting logic after creation and deletion

### CODE REVIEW

**Review by hildurhodd — APPROVED** [review]
LGTM

**Comment by paulomarquesc** [comment]
> 

The cleanup is not controlled through a variable, please change the readme comments to address the changes.

---
Refers to: README.md:24 in bea9d78. [](commit_id = bea9d78b2cfc75bbd6dc68f895a442a85a19d643, deletion_comment = False)

**Review by paulomarquesc — CHANGES_REQUESTED** [review]
:clock1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,22 @@
 ## Azure NetAppFiles SDK NFS 4.1 Sample for .NETCore Changelog
 
+[1.0.0 (2021-09-02)](#1.0.1 (2021-09-02))
```
Comment (paulomarquesc) [inline CHANGELOG.md]: >1.0.0 [](http://example.com/codeflow?start=1&length=5)

This version needs to be updated. #Resolved

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,22 @@
 ## Azure NetAppFiles SDK NFS 4.1 Sample for .NETCore Changelog
 
+[1.0.0 (2021-09-02)](#1.0.1 (2021-09-02))
 [1.0.0 (2019-10-09)](#1.0.0 (2019-10-09))
 
 # 1.0.0 (2019-10-09)
```
Comment (paulomarquesc) [inline CHANGELOG.md:7]: Version and date needs to be updated

**Review by paulomarquesc — APPROVED** [review]
:shipit:

**Pull request merged**